direct/src/particles/ParticleEffect.py

```diff
@@ -220,10 +220,13 @@ def softStop(self):
         for particles in self.getParticlesList():
             particles.softStop()
 
-    def softStart(self):
+    def softStart(self, firstBirthOffset = None):
         if self.__isValid():
             for particles in self.getParticlesList():
-                particles.softStart()
+                if firstBirthOffset is not None:
+                    particles.softStartOffset(boff = firstBirthOffset)
```

Comment:
Nitpick: there should not be spaces around a `=` in keyword argument lists according to PEP8 even though there should be spaces around it almost anywhere else.

Reply:
Ah, fair enough. My own style includes spaces, but I'll change this to reflect the preferred style for the engine, I intend.

Reply:
Don't worry too much about this particular issue; we're wildly inconsistent on this particular rule in the Panda codebase.  I wouldn't block the PR from merging on this matter.  We just try to default to PEP8 since it's the most popular style that is likely to be the most familiar to other developers.

Reply:
That's fair! But I have other changes to make anyway, as per your other comment, so I might as well include this one, too. ^_^